serotiny/io/image.py

```diff
@@ -84,6 +86,14 @@ def image_loader(
         If reading an ome zarr image, this specifies the image name
     """
 
+    # silence aicsimageio related warnings
+    warnings.filterwarnings(action="ignore", category=FutureWarning, module="ome_types")
+    logging.getLogger("xmlschema").setLevel(logging.ERROR)
```

Comment:
I'd rather wait for [this PR](https://github.com/AllenCellModeling/aicsimageio/pull/442) on `aicsimageio` to get merged rather than adding temporary changes here. And I think the second line isn't necessary anyway (I added it as an overkill)